src/Tests/Tests/epv_test_validate_php_functions.php

```diff
@@ -245,11 +246,10 @@ private function parseNodes(array $nodes)
 		{
 			foreach ($nodes as $node)
 			{
-				// Check if there is a class.
-				// If there is a class, there should be a namespace.
-				if ($node instanceof Class_ || $node instanceof InterfaceTest)
+				// Check if there is a class, interface or trait, there should be a namespace.
+				if ($node instanceof Class_ || $node instanceof InterfaceTest || $node instanceof Trait_)
 				{
-					$this->addMessage($this->isTest() ? Output::NOTICE : Output::ERROR, 'All files with a class or an interface should have a namespace');
```

Comment:
Please remove the then they in that sentence as it doesn't make much sense with it.

Reply:
> Please remove the then they in that sentence as it doesn't make much sense with it.

Is this alright `All files with a class, interface or trait should have a namespace` ???